### PR TITLE
Fix golint on circleci and GCP testing

### PR DESCRIPTION
**Pull request by justincormack**
Signed-off-by: Justin Cormack <justin.cormack@docker.com>

### CODE REVIEW

File: test/hack/test.yml [hunk 1]
```diff
@@ -13,6 +13,9 @@ onboot:
     command: ["/sbin/dhcpcd", "--nobackground", "-f", "/dhcpcd.conf", "-1"]
   - name: check-kernel-config
     image: linuxkit/test-kernel-config:1aaef970b5f70791d74d6f980ad38af4035948f8
+  - name: sleep
+    image: linuxkit/poweroff:afe8f7dd0d47a7991c54519b0f09124cb8c4e300
+    command: ["/bin/sleep", "120"]
```
Comment (rn) [inline test/hack/test.yml]: The `poweroff` package also takes an numerical argument for sleep. Maybe this needs bumping from 3s to something a bit more

Comment (justincormack) [reply test/hack/test.yml]: ah yes I forgot that; lets see if this has any effect anyway and will change that instead.

Comment (justincormack) [reply test/hack/test.yml]: ok that did fix it, will adjust....

**Comment by justincormack** [comment]
When I did `linuxkit run gcp` by hand the console connected quite late, so I guess they just cut some boot time somehow...

**Pull request merged**